pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/validator/ModelValidator.java

```diff
@@ -25,29 +25,7 @@
 
 package org.jenkinsci.plugins.pipeline.modeldefinition.validator;
 
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTAgent;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTBranch;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTBuildCondition;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTBuildConditionsContainer;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTBuildParameter;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTBuildParameters;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTEnvironment;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTInternalFunctionCall;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTLibraries;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTMethodCall;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTOption;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTOptions;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTPipelineDef;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTPostBuild;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTPostStage;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTStage;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTStages;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTStep;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTTools;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTTrigger;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTTriggers;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTWhen;
-import org.jenkinsci.plugins.pipeline.modeldefinition.ast.ModelASTWhenCondition;
+import org.jenkinsci.plugins.pipeline.modeldefinition.ast.*;
```

Comment:
ah, import reformat fun! I don't particularly care that much about that - I have IntelliJ set up to never do star imports personally, but this isn't going to cause my IDE to reformat this again in the future, so no harm, no foul overall. But I do know other people can be more resistant to "formatting only" changes like this. Just an FYI to watch out for in the future.

Reply:
fixed in a rebase to master, I don't want to lose the commentary